### PR TITLE
[IOTDB-597] Rename methods in Session

### PR DESCRIPTION
**Pull request by qiaojialin**
Rename  insertMultipleDeviceBatch to insertTables
Rename insertInBatch to insertRecords
add insertTablets in rpc.thrift
Rename insert to insertRecord in rpc
Rename insertBatch to insertTablet
Rename RowBatch to Tablet

### CODE REVIEW

**Comment by jixuan1989** [comment]
how about batchInsertInColumns() and batchInsertInRows()?

File: example/session/src/main/java/org/apache/iotdb/SessionExample.java [hunk 1]
```diff
@@ -125,20 +125,19 @@ private static void insertInBatch() throws IoTDBConnectionException, BatchExecut
...
+        session.insertRows(deviceIds, timestamps, measurementsList, valuesList);
         deviceIds.clear();
         measurementsList.clear();
         valuesList.clear();
         timestamps.clear();
       }
     }
 
-    session.insertInBatch(deviceIds, timestamps, measurementsList, valuesList);
+    session.insertRows(deviceIds, timestamps, measurementsList, valuesList);
```
Comment (liutaohua) [inline example/session/src/main/java/org/apache/iotdb/SessionExample.java:136]: don't forget to change the method name of ` this. insertInBatch()`

File: example/session/src/main/java/org/apache/iotdb/SessionExample.java [hunk 2]
```diff
@@ -211,10 +209,10 @@ private static void insertMultipleDeviceRowBatch()
     
     RowBatch rowBatch3 = schema3.createRowBatch("root.sg1.d3", 100);
     
-    Map<String, RowBatch> rowBatchMap = new HashMap<>();
-    rowBatchMap.put("root.sg1.d1", rowBatch1);
-    rowBatchMap.put("root.sg1.d2", rowBatch2);
-    rowBatchMap.put("root.sg1.d3", rowBatch3);
+    List<RowBatch> rowBatchMap = new ArrayList<>();
```
Comment (liutaohua) [inline example/session/src/main/java/org/apache/iotdb/SessionExample.java:212]: `rowBatchList`

File: example/session/src/main/java/org/apache/iotdb/SessionExample.java (hunk 1 above)
Comment (qiaojialin) [reply example/session/src/main/java/org/apache/iotdb/SessionExample.java]: fixed

File: example/session/src/main/java/org/apache/iotdb/SessionExample.java (hunk 2 above)
Comment (qiaojialin) [reply example/session/src/main/java/org/apache/iotdb/SessionExample.java]: fixed

File: example/hadoop/src/main/java/org/apache/iotdb/hadoop/tsfile/TsFileHelper.java [hunk 3]
```diff
@@ -52,39 +54,47 @@ public static void writeTsFile(String filePath) {
...
+            measurementSchema);
+        schemaList.add(measurementSchema);
       }
 
       // add measurements into TSFileWriter
       TsFileWriter tsFileWriter = new TsFileWriter(file, schema);
 
       // construct the row batch
-      RowBatch rowBatch = schema.createRowBatch(Constant.DEVICE_1);
+      Tablet tablet = new Tablet(Constant.DEVICE_1, schemaList);
```
Comment (HTHou) [inline example/hadoop/src/main/java/org/apache/iotdb/hadoop/tsfile/TsFileHelper.java:87]: // construct the tablet

File: server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java [hunk 4]
```diff
@@ -576,12 +576,12 @@ private boolean checkTTL(long time) {
   /**
    * insert batch to tsfile processor thread-safety that the caller need to guarantee
    *
-   * @param batchInsertPlan batch insert plan
+   * @param insertTabletPlan batch insert plan
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java:579]: thanks! fixed

File: session/src/test/java/org/apache/iotdb/session/IoTDBSessionIT.java [hunk 5]
```diff
@@ -206,14 +202,14 @@ public void testTestMethod()
     createTimeseries();
 
     // test insert batch
-    Schema schema = new Schema();
-    schema.registerTimeseries(new Path(deviceId, "s1"), new MeasurementSchema("s1", TSDataType.INT64, TSEncoding.RLE));
-    schema.registerTimeseries(new Path(deviceId, "s2"), new MeasurementSchema("s2", TSDataType.INT64, TSEncoding.RLE));
-    schema.registerTimeseries(new Path(deviceId, "s3"), new MeasurementSchema("s3", TSDataType.INT64, TSEncoding.RLE));
+    List<MeasurementSchema> schemaList = new ArrayList<>();
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply session/src/test/java/org/apache/iotdb/session/IoTDBSessionIT.java:205]: fixed

File: session/src/test/java/org/apache/iotdb/session/IoTDBSessionIT.java [hunk 6]
```diff
@@ -225,7 +221,7 @@ public void testTestMethod()
       values.add("1");
       values.add("2");
       values.add("3");
-      session.testInsert(deviceId, time, measurements, values);
+      session.testInsertRecord(deviceId, time, measurements, values);
     }
 
     // test insert row in batch
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply session/src/test/java/org/apache/iotdb/session/IoTDBSessionIT.java:227]: fixed

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/TsFileWriter.java [hunk 7]
```diff
@@ -208,22 +208,22 @@ private boolean checkIsTimeSeriesExist(TSRecord record) throws WriteProcessExcep
   /**
    * Confirm whether the row batch is legal.
    *
-   * @param rowBatch - a row batch responding multiple columns
+   * @param tablet - a row batch responding multiple columns
    * @return - whether the row batch has been added into RecordWriter legally
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply tsfile/src/main/java/org/apache/iotdb/tsfile/write/TsFileWriter.java:212]: fixed

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/TsFileWriter.java [hunk 8]
```diff
@@ -254,16 +254,16 @@ public boolean write(TSRecord record) throws IOException, WriteProcessException
   /**
    * write a row batch
    *
-   * @param rowBatch - multiple time series of one device that share a time column
+   * @param tablet - multiple time series of one device that share a time column
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply tsfile/src/main/java/org/apache/iotdb/tsfile/write/TsFileWriter.java:257]: fixed

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/IChunkGroupWriter.java [hunk 9]
```diff
@@ -51,14 +51,14 @@
   /**
    * receive a row batch, write it to timeseries writers
    *
-   * @param rowBatch
+   * @param tablet
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply tsfile/src/main/java/org/apache/iotdb/tsfile/write/chunk/IChunkGroupWriter.java:54]: fixed

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java [hunk 10]
```diff
@@ -46,49 +57,43 @@
...
-  /**
-   * total byte size that values occupies
-   */
-  private int valueOccupation = -1;
+  private int maxRowNumber;
 
   /**
    * Return a row batch with default specified row number. This is the standard
-   * constructor (all RowBatch should be the same size).
+   * constructor (all Tablet should be the same size).
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java:74]: fixed

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java [hunk 11]
```diff
@@ -46,49 +57,43 @@
...
    */
-  public RowBatch(String deviceId, List<MeasurementSchema> timeseries) {
+  public Tablet(String deviceId, List<MeasurementSchema> timeseries) {
     this(deviceId, timeseries, DEFAULT_SIZE);
   }
 
   /**
    * Return a row batch with the specified number of rows (maxBatchSize). Only
-   * call this constructor directly for testing purposes. RowBatch should normally
+   * call this constructor directly for testing purposes. Tablet should normally
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java:85]: fixed

File: tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java [hunk 12]
```diff
@@ -102,42 +107,42 @@ public RowBatch(String deviceId, List<MeasurementSchema> schemas, int maxBatchSi
   /**
    * Return the maximum number of rows for this row batch
    */
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply tsfile/src/main/java/org/apache/iotdb/tsfile/write/record/Tablet.java:109]: fixed

File: hive-connector/src/test/java/org/apache/iotdb/hive/TsFileTestHelper.java [hunk 13]
```diff
@@ -61,40 +64,42 @@ public static void writeTsFile(String filePath) {
...
+            measurementSchema);
+        schemaList.add(measurementSchema);
       }
 
       // add measurements into TSFileWriter
       TsFileWriter tsFileWriter = new TsFileWriter(file, schema);
 
       // construct the row batch
-      RowBatch rowBatch = schema.createRowBatch("device_1");
+      Tablet tablet = new Tablet("device_1", schemaList);
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply hive-connector/src/test/java/org/apache/iotdb/hive/TsFileTestHelper.java:78]: fixed

File: hadoop/src/test/java/org/apache/iotdb/hadoop/tsfile/TsFileTestHelper.java [hunk 14]
```diff
@@ -61,39 +64,42 @@ public static void writeTsFile(String filePath) {
...
+            measurementSchema);
+        schemaList.add(measurementSchema);
       }
 
       // add timeseries into TSFileWriter
       TsFileWriter tsFileWriter = new TsFileWriter(file, schema);
 
       // construct the row batch
-      RowBatch rowBatch = schema.createRowBatch("device_1");
+      Tablet tablet = new Tablet("device_1", schemaList);
```
[the remark this answers is not in the document]
Comment (qiaojialin) [reply hadoop/src/test/java/org/apache/iotdb/hadoop/tsfile/TsFileTestHelper.java:78]: fixed

File: example/hadoop/src/main/java/org/apache/iotdb/hadoop/tsfile/TsFileHelper.java (hunk 3 above)
Comment (qiaojialin) [reply example/hadoop/src/main/java/org/apache/iotdb/hadoop/tsfile/TsFileHelper.java]: fixed

**Pull request merged**